aiogram/utils/payload.py

```diff
@@ -59,7 +59,7 @@ def prepare_arg(value):
         return int((now + value).timestamp())
     if isinstance(value, datetime.datetime):
         return round(value.timestamp())
-    if isinstance(value, LazyProxy):
+    if isinstance(value, (LazyProxy, int, float)):
```

Comment:
we can add support for `decimal.Decimal` in here.

Reply:
Its support is not declared anywhere, so I think we don't need to add it :thinking:

Reply:
Why do we need to convert float and int to str?

Reply:
@JrooTJunior this is the current behavior in 2.x — casting everything (that wasn't yet prepared by `prepare_arg`) in `compose_data` to `str` before sending:
https://github.com/aiogram/aiogram/blob/d5a4c0c4afc6f03b9de19f7f68f852c9188cf072/aiogram/bot/api.py#L169